firmware/ec/test/include/test_ltc4015.h

```diff
@@ -0,0 +1,185 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+#ifndef _TEST_LTC4015_H
+#define _TEST_LTC4015_H
+
+#include "common/inc/global/Framework.h"
+#include "common/inc/ocmp_wrappers/ocmp_ltc4015.h"
+#include "drivers/GpioSX1509.h"
+#include "drivers/OcGpio.h"
+#include "fake/fake_GPIO.h"
+#include "fake/fake_I2C.h"
+#include "fake/fake_ThreadedISR.h"
+#include "helpers/array.h"
+#include "helpers/attribute.h"
+#include "helpers/memory.h"
+#include "inc/devices/ltc4015.h"
+#include <string.h>
+#include <math.h>
+#include "unity.h"
+
+#define LTC4015_INIT_BATTERY_VOLTAGE_LOW_LIMIT 16470
+#define LTC4015_INIT_BATTERY_VOLTAGE_HIGH_LIMIT 21844
+#define LTC4015_INIT_CHARGE_CURRENT_LOW_LIMIT 2047
+#define LTC4015_INIT_INPUT_VOLTAGE_LOW_LIMIT 9830
+#define LTC4015_INIT_INPUT_CURRENT_HIGH_LIMIT 23892
+#define LTC4015_INIT_INPUT_CURRENT_LIMIT_SETTING 76
+/* ======================== Constants & variables =========================== */
+
+extern const OcGpio_FnTable GpioSX1509_fnTable;
+
+typedef enum LTC4015Status {
```

Comment:
can we remove these enums from ocmp_ltc4015.c and the test file and move it into the ltc4015.h ?

Reply:
Done